### DISCUSSION TITLE
Concerned Person / Receiver and direction (in/out) of document

### DISCUSSION THREAD

**Discussion by totti4ever** [jonaswinkler/paperless-ng#62] [Feature Requests]
_from #53:_

> 3. Concerned person
>> This is the correspondent of a document. This works in both ways.

> 5. Direction of item
>> It seems you want separate fields for sender and recipient.

I pulled 3 and 5 together as they should be treated together most probably.
I would like to specify both sides, sender and recipient for every item. But, yeah - if I would filter on `sender = me`, I actually had filtered on outgoing, so that would help

---

I'd like to have this idea as separate discussion as well as it really helped to organize things; knowing that one could solve it with nested tags (which is good for other things, but here I think it would be too technical)

**Comment by jonaswinkler** [comment]
I've just had a quick look at docspell and now I see where that idea comes from. Would a simple selection field "direction = incoming|outgoing" and a filter option "any|incoming|outgoing" be enough? Do we need another value "unspecified" (or similar) for documents that aren't exactly addressed at you? (I don't know, manuals, information material, ...)

**Reply by totti4ever** [reply]
Hey, basically I'd reckon a file could always be assigned to incoming or outgoing. At least when you consider everything not outgoing to be incoming in the meaning that you didn't create it yourself like manuals. So, yes that could be an option.

Still, comparing it to the other attributes a "---" (unset) entry like for correspondents might be a better option. Then again, you also have to set a document date.

So, in case it helps I think it's feasible to do without an _unset_ status and have incoming as default. If anybody doesn't care for the direction, he/she won't filter on it anyway.

Did you have a look at the concerned person as well? I would use that for keeping track if a letter is meant for me, my wife or one of the kids. As soon as we had nested tags, I could also use them, of course!

**Reply by muCruiser** [reply]
Could  correspondents field be changed to a mulit-select like tags.
In my point of view this would solve the topic as well, because you could select a sender and a recipient

**Reply by jonaswinkler** [reply]
That's not a simple change, since many parts of paperless rely on each document having exactly one correspondent.
